### ISSUE TITLE
Workflow file is not in sync with GFguide requirements, causing release failure

### ISSUE THREAD

**Issue by n8willis** [googlefonts/googlefonts-project-template#182]
The issue:

The latest update to the workflow file:
https://github.com/googlefonts/googlefonts-project-template/blame/ca4b21bee0745406b28c1a647d02802634e03a1a/.github/workflows/build.yaml#L91

has hardcoded-in locations for ARTICLE.en_us.html and DESCRIPTION.en_us.html. It looks for ARTICLE to be in the `documentation/` directory, but the guide tells designers that it should be in `article/` (see https://googlefonts.github.io/gf-guide/article.html#the-article ).

Therefore the "Create and populate release" job fails if the designer follows the guide.

The job *also* fails if DESCRIPTION is not in `documentation/`, even though the guide says that DESCRIPTION is deprecated, and even though older versions of the template repo had DESCRIPTION in the project root. The guide also doesn't say where, specifically, the DESCRIPTION file needs to be, although it says "This file follows the same requirements as the Article file" which implies `article/`.

Solution: either the workflow or the guide needs changing so that they match. 

Secondarily, if DOCUMENTATION is considered legacy now, then its absence shouldn't cause the job to fail.

**Comment by n8willis** [comment]
I am politely suggesting that you know what it's "referring to" because, independently of reading the page text, you already know that some other specific meaning is intended. For a person who comes in from the outside, that meaning is not at all communicated.

**Comment by simoncozens** [comment]
I completely agree that the guide needs clarifying. I don't think there's a bug in googlefonts-project-template, though.
